tests/integration/cmd_pref_config_test.go

```diff
@@ -51,6 +51,21 @@ var _ = Describe("odo preference and config command tests", func() {
 		})
 	})
 
+	When("running odo config view", func() {
+		It("should pass if devfile is present in current dir", func() {
+			helper.Chdir(commonVar.Context)
+			cmpName := helper.RandString(6)
+			helper.Cmd("odo", "create", cmpName, "--devfile", helper.GetExamplePath("source", "devfiles", "nodejs", "devfile.yaml")).ShouldPass()
+			out := helper.Cmd("odo", "config", "view").ShouldPass().Out()
+			helper.MatchAllInOutput(out, []string{"runtime", "Memory: 1024Mi"})
+		})
```

Comment:
IMO, it's sufficient to have failure test in this case because other parts of this test file are executing `odo config view` in some form, and hence the success case is being implicitly tested. 

I'm more of "explicit is better than implicit" person, but every extra integration test adds to our already slow integration test suite, so I'm fine with implicit nature here.

Reply:
I think we should have this case as this is different from what we are testing in the other test case. Also, this test case will not take much time to execute, for me, it passed withing 6-8 seconds.